publication.gradle

```diff
@@ -0,0 +1,29 @@
+apply from: '../target.gradle'
+final target = currentTarget()
+
+kotlin {
+    final taskConfigurationMap = [
+            'kotlinMultiplatform': target.isMeta(),
+            'metadata'           : target.isMeta(),
+            'jvm'                : target.isJvm(),
+            'js'                 : target.isJvm(),
```

Comment:
Why isJvm for JS? Looks like we need isJs, or just rename it to something common.